db/migrate/20250503112337_add_no_meeting_days_to_courses.rb

```diff
@@ -0,0 +1,5 @@
+class AddNoMeetingDaysToCourses < ActiveRecord::Migration[7.0]
+  def change
+    add_column :courses, :no_meeting_days, :boolean, default: false
```

Comment:
This should be implemented as a course flag (similar to `no_sandboxes` and other flags that control course behavior), rather than adding a database column.